modules/core/arrow-core-data/src/test/kotlin/arrow/core/SortedMapKTest.kt

```diff
@@ -32,6 +45,54 @@ class SortedMapKTest : UnitSpec() {
         SortedMapK.traverse<String>(),
         SortedMapK.functor<String>(),
         { a: Int -> sortedMapOf("key" to a).k() },
-        EQ))
+        EQ),
+      SemialignLaws.foldablelaws(SortedMapK.semialign<String>(),
+        Gen.sortedMapK(Gen.string(), Gen.int()) as Gen<Kind<SortedMapKPartialOf<String>, Int>>,
```

Comment:
This can be avoided if we redefine the generator.

`fun <K : Comparable<K>, V> Gen.Companion.sortedMapK(genK: Gen<K>, genV: Gen<V>): Gen<SortedMapKOf<K, V>> =`

Reply:
tried that out.  the types do not match anymore in the  MonoidLaws.laws() function call a few lines above.